### ISSUE TITLE
Slides not appearing on the display in Presenter Mode

### ISSUE THREAD

**Issue by habara-k** [hiroppy/fusuma#139]
## Info

- macOS Mojave version: 10.14.5
- Node Version: no relation
- Browser Name and Version: Chrome version: 75.0.3770.100

## Type

- [x] bug
- [ ] feature request

## For Bugs

I tried to use the Presenter Mode in the [DEMO](https://hiroppy.github.io/fusuma/intro), but the display only shows a blank screen.

### Expected Behavior

Slides appear on the connected display in Presenter mode

### Actual Behavior

The display only shows a blank screen.

### How can we reproduce the behavior?

1. Connect your PC to an external display
2. Access [What's Fusuma?](https://hiroppy.github.io/fusuma/intro) in Chrome
3. Open Sidebar(click the bottom right button(三))
4. Click the PC monitor icon
5. Select the display connected in step 1
6. The display does not show the slides

**Comment by hiroppy** [comment]
Thank you for the report. I fixed this issue. You can check https://hiroppy.github.io/fusuma/intro/#slide=1.

**Comment by habara-k** [comment]
Thank you for your prompt response !

However, after checking, the operation is not stable...

After launching Presenter Mode, the slide is displayed once or twice, but it immediately becomes a blank screen...

Is there anything to do to reflect your fix?

![presenter_mode](https://user-images.githubusercontent.com/34413567/60691720-bc5ac500-9f0c-11e9-8ebc-85bf4076ac5d.gif)

**Comment by hiroppy** [comment]
Currently, I don't have another display, so I cannot check Presentation API. 
Please submit error logs.

**Comment by matamatak** [comment]
@hiroppy 
I tried to reproduce the behavior.

1. Connect your PC to an external display
2. Access What's Fusuma? in Chrome
3. Open Sidebar(click the bottom right button(三))
4. Click the PC monitor icon
5. Select the display 1  (MacBook Air Display)

![スクリーンショット 2019-07-05 14 32 19](https://user-images.githubusercontent.com/39780486/60700226-0b672100-9f32-11e9-8c24-f32724878e8a.png)

6. The window showed but does not show the slides
<img width="1148" alt="スクリーンショット 2019-07-05 14 39 48" src="https://user-images.githubusercontent.com/39780486/60700400-c68fba00-9f32-11e9-993e-801f3f46cdaf.png">

7. Check console
```
vendor.10.acf9a91b990fe3b7fe96.bundle.js:32 

SyntaxError: Unexpected token o in JSON at position 1
    at JSON.parse (<anonymous>)
    at v (presenter.view.7.bd00434fa5ccc4e23352.bundle.js:1)
    at presenter.view.7.bd00434fa5ccc4e23352.bundle.js:1
    at ma (vendor.10.acf9a91b990fe3b7fe96.bundle.js:32)
    at vendor.10.acf9a91b990fe3b7fe96.bundle.js:32
    at f (vendor.10.acf9a91b990fe3b7fe96.bundle.js:46)
    at p (vendor.10.acf9a91b990fe3b7fe96.bundle.js:46)
    at MessagePort.D.port1.onmessage (vendor.10.acf9a91b990fe3b7fe96.bundle.js:46)
```

**Comment by hiroppy** [comment]
@matamatak Thank you for checking! I think we use Presentation API, so we cannot use localstorage. ok, I'll comment out these lines.

**Comment by hiroppy** [comment]
Hey, guys. I deployed fixed samples just now. Could you check them? https://hiroppy.github.io/fusuma/intro/

**Comment by matamatak** [comment]
@hiroppy 
Work!👌
Thanks.

**Comment by hiroppy** [comment]
I've released fusuma@1.9.4. Thank you so much.
